### DISCUSSION TITLE
How to resolve Permission Denied Error when we specify report name - Docker run --report=filename

### DISCUSSION THREAD

**Discussion by prakashsivakumars** [zricethezav/gitleaks#697] [Q&A]
I have been trying in Window 10 - Powershell in admin mode and I get the below error. I could not get the json or csv file created.
Any help would be greatly appreciated!!

PS C:\projects\repo-develop> docker run -v ${PWD}:/repo-develop zricethezav/gitleaks:latest --path="/repo-develop" --no-git --report=C:\gilleaksreport\healthmatter-develop.json --debug
time="2021-11-17T18:56:13Z" level=info msg="scan time: 1 minute 29 seconds 69 milliseconds 839 microseconds"
time="2021-11-17T18:56:13Z" level=warning msg="leaks found: 9"
time="2021-11-17T18:56:13Z" level=error msg="open C:\\gilleaksreport\\repo-develop.json: permission denied"

**Comment by maltemorgenstern** [comment]
Hey @prakashsivakumars,
when running inside a docker container you can only specify paths **inside** the container as report paths.
With your current setup gitleaks (inside the linux container) is trying to create a file using a Windows host path - which does not work.

Instead use a linux path to a volume so that you can save the report on your host:
```powershell
PS C:\projects\repo-develop> docker run -v ${PWD}:/repo-develop zricethezav/gitleaks:latest --path="/repo-develop" --no-git --report=/report-develop/healthmatter-develop.json --debug
```

If you don't want to save the report inside the `repo-develop` folder you can configure an additional volume for the report.

I hope this helps you.
Regards, Malte
